Habitica/src/main/java/com/habitrpg/android/habitica/callbacks/SkillCallback.java

```diff
@@ -37,6 +37,9 @@ public void call(SkillResponse skillResponse) {
             this.user.setFlags(user.getFlags());
         }
         if (user.getStats() != null) {
+            usedSkill.xp = user.getStats().getExp() - this.user.getStats().getExp();
+            usedSkill.hp = user.getStats().getHp() - this.user.getStats().getHp();
+            usedSkill.gold = user.getStats().getGp() - this.user.getStats().getGp();
```

Comment:
I don't think adding the temporary feedback variables to the skills object is the right approach. Why not put them as additional attributes on the SkillUsedEvent object?